augly/video/helpers/ffmpeg.py

```diff
@@ -28,11 +30,42 @@ def combine_frames_and_audio_to_file(
             "a directory"
         )
 
-    video_stream = ffmpeg.input(raw_frames, pattern_type="glob", framerate=framerate)
-    video_stream = video_stream.filter(
-        "pad", **{"width": "ceil(iw/2)*2", "height": "ceil(ih/2)*2"}
-    )
-    merge_video_and_audio(video_stream, audio, output_path)
+    with tempfile.TemporaryDirectory() as tmpdir:
+        temp_video_path = os.path.join(tmpdir, "out.mp4")
+        writer = WriteGear(output_filename=temp_video_path, logging=True)
+        ffmpeg_command = [
+            "-y",
+            "-framerate",
+            str(framerate),
+            "-pattern_type",
+            "glob",
+            "-i",
+            raw_frames,
+            "-c:v",
+            "libx264",
+            "-pix_fmt",
+            "yuv420p",
+            "-preset",
+            "ultrafast",
+            temp_video_path,
+        ]
+        writer.execute_ffmpeg_cmd(ffmpeg_command)
+        writer.close()
+        temp_padded_video_path = os.path.join(tmpdir, "out1.mp4")
+        writer = WriteGear(output_filename=temp_padded_video_path, logging=True)
+        ffmpeg_command = [
+            "-y",
+            "-i",
+            temp_video_path,
+            "-vf",
+            "pad=width=ceil(iw/2)*2:height=ceil(ih/2)*2",
+            "-preset",
+            "ultrafast",
+            temp_padded_video_path,
+        ]
+        writer.execute_ffmpeg_cmd(ffmpeg_command)
+        writer.close()
+        merge_video_and_audio(temp_padded_video_path, audio, output_path)
```

Comment:
Our internal linter is giving us an error on this line: "Incompatible parameter type [6]: Expected `Optional[str]` for 2nd positional only parameter to call `merge_video_and_audio` but got `Union[None, io.BytesIO, str]`."

Can you change the type of `audio` on line 22 to `Optional[str]` so it aligns with the type expected by `merge_video_and_audio()`? I checked & this function is only called from [`video/utils.py`](https://github.com/facebookresearch/AugLy/augly/video/utils.py) and always with audio as `None` or a str, so it should be fine to change this.